src/test/java/seedu/address/logic/commands/CancelClearCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalRecipes.getTypicalRecipeBook;
+
+import org.junit.jupiter.api.Test;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for.
+ * {@code ClearCommand}.
+ */
+public class CancelClearCommandTest {
+    private final Model model = new ModelManager(getTypicalRecipeBook(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalRecipeBook(), new UserPrefs());
```

Comment:
I think it's better to use RecipeBookModelManager since that's what the rest of our production code uses... for now at least we'll standardize and rename it to ModelManager in the refactoring phase!

```suggestion
    private final Model model = new RecipeBookModelManager(getTypicalRecipeBook(), new UserPrefs());
    private final Model expectedModel = new RecipeBookModelManager(getTypicalRecipeBook(), new UserPrefs());
```